api/client/samples/crop_models/brazil_soybeans.ipynb

```diff
@@ -42,10 +42,11 @@
      "name": "stdout",
```

Comment:
<p>looks like some strange values are appearing in 2000 or 2001 now</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>